examples/simple_solution.xml

```diff
@@ -0,0 +1,99 @@
+<database
+        xmlns="http://calphad.org/thermml/0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:xi="http://www.w3.org/2001/XInclude"
+        xsi:schemaLocation="http://calphad.org/thermml/0.0 ../schema/thermml-schema.xsd"
+
+        name="Example Simple Solution Database">
+
+    <description>
+        This is an example thermodynamic database for demonstrating the ThermML schema for simple solutions.
+    </description>
+
+    <nonChemicalPotentials>
+        <nonChemicalPotential id="potT" symbol="T" units="K"/>
+        <nonChemicalPotential id="potP" symbol="P" units="bar"/>
+        <nonChemicalPotential id="potB" symbol="B" units="T"/>
+    </nonChemicalPotentials>
+
+    <systemComponents>
+        <systemComponent symbol="Va" />
+        <systemComponent symbol="C"  refstate="GRAPHITE" molar_mass="12.011"  h298="1054.0"   s298="5.7423"  />
+        <systemComponent symbol="Fe" refstate="BCC_A2"   molar_mass="55.847"  h298="4489.0"   s298="27.2797" />
+        <systemComponent symbol="Mn" refstate="CBCC_A12" molar_mass="54.9380" h298="4995.696" s298="32.2206" />
+    </systemComponents>
+
+    <phases>
+        <phase name="FCC_A1" alias="Austenite" state="solid" xsi:type="SimpleSolutionPhaseType">
+            <structure spacegroup="225" prototype="NaCl">
+                <sublattices multiplicities="1.0 1.0">
+                    <site constituents="Fe Mn" wyckhof="4a"/>
+                    <site constituents="C Va" wyckhof="4b"/>
+                </sublattices>
+            </structure>
+
+            <property xsi:type="G">
+                <constituents>Fe:Va</constituents> 
+                <reference>91Din</reference>
+                <expr>Fe#FCC_A1</expr>
```

Comment:
`Fe#FCC_A1`, `C#HSER`, are just names, correct? You could have named them something else in the corresponding `<function>`, right? Just confirming my understanding that there's nothing special about the `#` symbol here.

Reply:
Yes, they are just names. In the FactSage universe, this is a typical (but not mandatory) way of writing the functions. Whatever would be in a TDB or other type of database works just as well.